src/Storefront/Resources/snippet/en_GB/storefront.en-GB.json

```diff
@@ -278,8 +278,14 @@
     "addressesSetAsDefaultBillingAction": "Use as default billing address",
     "addressesTitleCurrentShippingAddress": "Current shipping address",
     "addressesTitleCurrentBillingAddress": "Current billing address",
+    "addressesTitleShippingAddress": "Shipping",
+    "addressesTitleBillingAddress": "Billing",
```

Comment:
In the german you have shipping address and billing address, in the english you only have shipping and billing. is this intentional?